sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_repository_client.py

```diff
@@ -104,19 +104,20 @@ async def get_properties(self, **kwargs: Dict[str, Any]) -> RepositoryProperties
     @distributed_trace_async
     async def get_registry_artifact_properties(
```

Comment:
out of curiosity, is there a reason you didn't rename the `registry_artifact` methods to `artifact_manifest`?

Reply:
I left this alone because in a separate PR (#18392) I re-shape the entire surface of the API. This method goes away and is put on the a `RegistryArtifact` object and the `ContainerRepositoryClient` also is renamed to `ContainerRepository` with fewer methods that are specific to interactions with just a repository.